application/application.go

```diff
@@ -0,0 +1,166 @@
+package application
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/config"
+	"github.com/josephroberts/edge-node-manager/database"
+	"github.com/josephroberts/edge-node-manager/device"
+	"github.com/josephroberts/edge-node-manager/helper"
+	"github.com/josephroberts/edge-node-manager/proxyvisor"
+	"github.com/josephroberts/edge-node-manager/radio"
+)
+
+type ApplicationInterface interface {
```

Comment:
`ApplicationInterface` is redundant. `Interface` will do here. For one interfaces in go are not explicitly referenced, for another if a package that uses this needs to reference it, it'll reference it as `Application.Interface`. `Application.ApplicationInterface` is a bit of a mouthful.